layers/+lang/python/layers.el

```diff
@@ -0,0 +1,14 @@
+;;; layers.el --- Python Layer declarations File for Spacemacs
+;;
+;; Copyright (c) 2012-2019 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(when (and (boundp 'python-backend)
+           (eq python-backend 'lsp))
+  (configuration-layer/declare-layer 'lsp))
```

Comment:
This is essentially the same thing the `c-c++` layer does: conditionally depend on `lsp` when the user has enabled support for it.

https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Blang/c-c%2B%2B/layers.el